OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidEntryPointsFinder.scala

```diff
@@ -0,0 +1,54 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+package android
+
+import scala.collection.mutable.ArrayBuffer
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+import org.opalj.br.Method
+import org.opalj.br.MethodDescriptor
+import org.opalj.br.ReferenceType
+import org.opalj.br.analyses.SomeProject
+import org.opalj.br.analyses.cg.EntryPointFinder
+import org.opalj.br.android.AndroidManifest
+import org.opalj.br.android.AndroidManifestKey
+import org.opalj.tac.fpcf.analyses.MethodDescription
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of launcher Activity Clases as entry points.
+ * An activity is a launcher activity if it contains an intent filter with action "android.intent.action.MAIN"
+ * and category "android.intent.category.LAUNCHER". Requires Android Manifest to be loaded.
+ *
+ * @author Tom Nikisch
+ *         Julius Naeumann
+ */
+object AndroidEntryPointsFinder extends EntryPointFinder {
+    val configKey = "org.opalj.tac.cg.android.AndroidEntryPointsFinder.entryPoints"
```

Comment:
Add blank lines around here to make it more readable

Reply:
Why was this marked as resolved? The val and def still stick together here impeding readability.